### PR TITLE
Additional information on how to configure automatic image signature import 

### PR DESCRIPTION
**Pull request by mburke5678**
[Describe how to configure automatic image signature import](https://github.com/openshift/openshift-docs/pull/5485)

### CODE REVIEW

**Comment by mburke5678** [comment]
@mfojtik @zhouying7780 @thomasmckay
How does this look? 
I brought down some information from the github file and moved the link to the end to de-emphasize it.
@mburke5678

**Comment by mburke5678** [comment]
[rev_history]
|xref:../admin_guide/image_signatures.adoc#admin-guide-image-signatures[Image Signatures]
|Added new section on xref:../admin_guide/image_signatures.adoc#importing-signatures-from-sigstore[Importing Image Signatures Automatically].
%

**Comment by wanghaoran1988** [comment]
LGFM

**Comment by mburke5678** [comment]
@mfojtik  @thomasmckay PTAL

**Comment by mburke5678** [comment]
@mfojtik  @thomasmckay  PTAL

**Comment by thomasmckay** [comment]
ack, thanks!

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL
This is a follow-up to https://github.com/openshift/openshift-docs/pull/5485

The original PR was written by Michal Fojtik and linked to a GitHub page[1]. I re-org'ed the original, added info from the GitHub page.

[1] https://github.com/containers/image/blob/master/docs/registries.d.md

**Comment by mburke5678** [comment]
No rev history needed

File: admin_guide/image_signatures.adoc [hunk 1]
```diff
@@ -252,23 +252,35 @@ link:https://github.com/containers/image/blob/master/docs/atomic-signature.md#th
 === Importing Image Signatures Automatically from Signature Stores
 
 {product-title} can automatically import image signatures if a signature
-store is configured on all {product-title} master nodes through the _registries
-configuration directory_ located in the *_/etc/containers/registries.d_*. 
+store is configured on all {product-title} master nodes through the registries
+configuration directory. 
 
-The default registries configuration directory is the *_/etc/containers/registries.d/default.yaml_* file. 
+The registries configuration directory contains configuration
```
Comment (nengard) [inline admin_guide/image_signatures.adoc]: contains *the* configuration

File: admin_guide/image_signatures.adoc [hunk 2]
```diff
@@ -252,23 +252,35 @@ link:https://github.com/containers/image/blob/master/docs/atomic-signature.md#th
...
 
 {product-title} can automatically import image signatures if a signature
-store is configured on all {product-title} master nodes through the _registries
-configuration directory_ located in the *_/etc/containers/registries.d_*. 
+store is configured on all {product-title} master nodes through the registries
+configuration directory. 
 
-The default registries configuration directory is the *_/etc/containers/registries.d/default.yaml_* file. 
+The registries configuration directory contains configuration 
+for various registries (servers storing remote container images),
```
Comment (nengard) [inline admin_guide/image_signatures.adoc]: no comma

File: admin_guide/image_signatures.adoc [hunk 3]
```diff
@@ -252,23 +252,35 @@ link:https://github.com/containers/image/blob/master/docs/atomic-signature.md#th
...
 {product-title} can automatically import image signatures if a signature
-store is configured on all {product-title} master nodes through the _registries
-configuration directory_ located in the *_/etc/containers/registries.d_*. 
+store is configured on all {product-title} master nodes through the registries
+configuration directory. 
 
-The default registries configuration directory is the *_/etc/containers/registries.d/default.yaml_* file. 
+The registries configuration directory contains configuration 
+for various registries (servers storing remote container images), 
+and for content stored in them. The single directory ensures that the configuration
```
Comment (nengard) [inline admin_guide/image_signatures.adoc]: for *the*  content

File: admin_guide/image_signatures.adoc [hunk 4]
```diff
@@ -252,23 +252,35 @@ link:https://github.com/containers/image/blob/master/docs/atomic-signature.md#th
...
-store is configured on all {product-title} master nodes through the _registries
-configuration directory_ located in the *_/etc/containers/registries.d_*. 
+store is configured on all {product-title} master nodes through the registries
+configuration directory. 
 
-The default registries configuration directory is the *_/etc/containers/registries.d/default.yaml_* file. 
+The registries configuration directory contains configuration 
+for various registries (servers storing remote container images), 
+and for content stored in them. The single directory ensures that the configuration 
+does not have to be provided in command-line options for each command,
```
Comment (nengard) [inline admin_guide/image_signatures.adoc]: no comma needed

File: admin_guide/image_signatures.adoc [hunk 5]
```diff
@@ -252,23 +252,35 @@ link:https://github.com/containers/image/blob/master/docs/atomic-signature.md#th
...
-configuration directory_ located in the *_/etc/containers/registries.d_*. 
+store is configured on all {product-title} master nodes through the registries
+configuration directory. 
 
-The default registries configuration directory is the *_/etc/containers/registries.d/default.yaml_* file. 
+The registries configuration directory contains configuration 
+for various registries (servers storing remote container images), 
+and for content stored in them. The single directory ensures that the configuration 
+does not have to be provided in command-line options for each command, 
+and so that it can be shared by all users of containers/image.
```
Comment (nengard) [inline admin_guide/image_signatures.adoc]: all *the* users of *the* containers

File: admin_guide/image_signatures.adoc [hunk 6]
```diff
@@ -252,23 +252,35 @@ link:https://github.com/containers/image/blob/master/docs/atomic-signature.md#th
...
 
 ----
 docker:
   registry.access.redhat.com:
-    sigstore: https://access.redhat.com/webassets/docker/content/sigstore
+    sigstore: https://access.redhat.com/webassets/docker/content/sigstore <1>
 ----
 
-Note that all signatures imported automatically by {{product-title}} will be
+<1> Defines the URL of an signature store. This URL is used for reading existing signatures.
```
Comment (nengard) [inline admin_guide/image_signatures.adoc]: of *a* (not *an*) signature store

File: admin_guide/image_signatures.adoc [hunk 7]
```diff
@@ -252,23 +252,35 @@ link:https://github.com/containers/image/blob/master/docs/atomic-signature.md#th
...
+[NOTE] 
+====
+Signatures imported automatically by {product-title} will be
 _unverified_ by default and will have to be verified by image administrators.
+====
 
 For more details about the registries configuration directory, see
 link:https://github.com/containers/image/blob/master/docs/registries.d.md[Registries Configuration Directory] 
-in the *containers/image* library documentation.
+in *containers/image* library documentation.
```
Comment (nengard) [inline admin_guide/image_signatures.adoc]: this should be 

"see the containers/image library"

**Review by ncbaratta — CHANGES_REQUESTED** [review]
A few fixes needed

File: admin_guide/image_signatures.adoc (hunk 7 above)
Comment (mburke5678) [reply admin_guide/image_signatures.adoc]: Maybe I should remove the whole phrase and go with _see
link:https://github.com/containers/image/blob/master/docs/registries.d.md[Registries Configuration Directory]._ ??

**Comment by mburke5678** [comment]
[rev_history]
|xref:../admin_guide/image_signatures.adoc#admin-guide-image-signatures[Image Signatures]
|Added information on xref:../install_config/build_defaults_overrides.adoc#importing-signatures-from-sigstore[configuring automatic image signature import].
%

**Pull request merged**